config/v1/types_scheduling.go

```diff
@@ -93,6 +89,22 @@ var (
 	NoScoring SchedulerProfile = "NoScoring"
 )
 
+// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles
+type ProfileCustomizations struct {
+	// experimentalDynamicResourceAllocation enables/disables dynamic resource allocation feature.
+	// +kubebuilder:default=Disabled
```

Comment:
Is this default value likely to change over time?

Reply:
Less likely it's going to change.

Reply:
Less likely 🤔 In general for configuration APIs like this, we tend to not have a default value, and instead allow the empty string, and then say that the empty string (or we normally call it omitted), means `no opinion` and that the default is subject to change over time. You probably want to follow that pattern here too

Reply:
Done